### PR TITLE
Chat Demo updated with Meeting Functionality 

### PR DESCRIPTION
**Pull request by kmai-amazon**
**Issue #:**
We have provided AWS Chime customers with both a Chat demo and a Meeting demo. However, some customers are having difficulty combining the two into a single serverless client. So we've updated the Chat demo to include meetings functionality

**Description of changes:**
- Updated the apps/chat demo to support video and audio meetings
- Updated CF template in the apps/chat demo to deploy additional endpoints to the APIG to act as a meeting server with CreateMeeting, CreateAttendee, and EndMeeting functionality

**Testing**

1. How did you test these changes?
The changes were tested locally by following the steps in the updated README and in the blog post.
2. Can these changes be tested using one of the demo application? If yes, which demo application can be used to test it?
This change is a demo application

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by simmkyu — CHANGES_REQUESTED** [review]
- Replace `Apache-2.0` to `MIT-0` in all files.
- Recommend moving the Lambda code from the CF template to individual files.
- PR includes a mix of JavaScript (js, jsx) and TypeScript (ts, tsx) files. I recommend using one language.

File: apps/chat/package.json
Comment (kmai-amazon) [reply apps/chat/package.json:7]: Removed

File: apps/chat/src/backend/serverless/template.yaml
Comment (kmai-amazon) [reply apps/chat/src/backend/serverless/template.yaml:92]: Removed the import in ChimeAPI.js since it is not used

File: apps/chat/src/backend/serverless/template.yaml
Comment (kmai-amazon) [reply apps/chat/src/backend/serverless/template.yaml:104]: Done

File: apps/chat/src/components/ActivityBar/index.tsx
Comment (kmai-amazon) [reply apps/chat/src/components/ActivityBar/index.tsx:2]: Done

File: apps/chat/src/backend/serverless/template.yaml
Comment (kmai-amazon) [reply apps/chat/src/backend/serverless/template.yaml:860]: Yes, we are only allowing the localhost origin as that is what is specified in the demo instructions.

File: apps/chat/src/backend/serverless/template.yaml
Comment (kmai-amazon) [reply apps/chat/src/backend/serverless/template.yaml:89]: The issue with moving the lambda source code to a separate file is that there is no automatic way (besides switching to CDK) to upload the lamdba sources to S3 for CF to use. Until we migrate this to CDK, by keeping all the lambda code in the CF template, it will always be up to date.

**Comment by kmai-amazon** [comment]
Addressed all feedback. As for the mix of JavaScript and TypeScript, the original Chat demo was purely in JavaScript while the new code borrowed from the meetings demo is in TypeScript. Although ideally, this should all be in TypeScript, this is not a no-op task to convert all the JavaScript files.

File: apps/chat/src/backend/serverless/template.yaml
Comment (kmai-amazon) [reply apps/chat/src/backend/serverless/template.yaml:136]: Updated

File: apps/chat/src/backend/serverless/template.yaml
Comment (kmai-amazon) [reply apps/chat/src/backend/serverless/template.yaml:153]: Updated

File: apps/chat/src/backend/serverless/template.yaml
Comment (kmai-amazon) [reply apps/chat/src/backend/serverless/template.yaml:827]: For the sake of the blog and keeping all the sources up-to-date, unless we switch to CDK, we would have to upload these to an S3 bucket and reference them in the CF. You're absolutely right that as separate files, we would be able to better run formatters and read the code but for the deployment, it would make keeping a consistent version for the CF templates and lamdbda handlers more difficult.

**Comment by simmkyu** [comment]
> Addressed all feedback. As for the mix of JavaScript and TypeScript, the original Chat demo was purely in JavaScript while the new code borrowed from the meetings demo is in TypeScript.

Thanks for your comments. Other that that, PR looks good to me.

**Review by simmkyu — APPROVED** [review]

**Comment by michhyun1** [comment]
@kmai-amazon should this be merged?

**Comment by kmai-amazon** [comment]
We are still waiting for approval to release alongside the blog.

**Pull request merged**
